src/components/NavBar/EditorDesignName.tsx

```diff
@@ -36,6 +40,21 @@ const DesignName = () => {
     );
   };
 
+  const saveNameChange = () => {
+    if (newDesignName.trim().length > DESIGN_NAME_MAX_CHAR_LENGTH) {
```

Comment:
The name length check seems like part of the check design name function. It seems like the length check can be moved into the checkDesignName().